plugins/targetlocker/inmemory/inmemory.go

```diff
@@ -170,8 +170,10 @@ func broker(lockRequests, unlockRequests, checkLocksRequests <-chan *request, do
 type InMemory struct {
 	lockRequests, unlockRequests, checkLocksRequests chan *request
 	done                                             chan struct{}
-	// timeout set on each lock request
-	timeout time.Duration
+	// initialTimeout set on each lock request
+	initialTimeout time.Duration
```

Comment:
this is probably `lockTimeout` rather than `initialTimeout`, as it is used with `Lock` as opposed to `RefreshLock`